tests/ui/extern/extern-const.fixed

```diff
@@ -6,12 +6,10 @@
 
 //@ run-rustfix
 //@ compile-flags: -g
-#![feature(rustc_private)]
-extern crate libc;
 
 #[link(name = "rust_test_helpers", kind = "static")]
 extern "C" {
-    static rust_dbg_static_mut: libc::c_int; //~ ERROR extern items cannot be `const`
+    static rust_dbg_static_mut: i32; //~ ERROR extern items cannot be `const`
```

Comment:
This is defines as `int` on the C side, so using `i32` will be incorrect on some architectures. Maybe use `int32_t` on the C side?

https://github.com/rust-lang/rust/blob/5dcb678ad8dc23a518f9ddf907e6a08de923d05e/tests/auxiliary/rust_test_helpers.c#L242-L243

Reply:
Fixed by using `ffi::c_int`.